src/webgpu/api/validation/createPipelineLayout.spec.ts

```diff
@@ -102,3 +102,14 @@ g.test('number_of_bind_group_layouts_exceeds_the_maximum_value').fn(async t => {
     t.device.createPipelineLayout(badPipelineLayoutDescriptor);
   });
 });
+
+g.test('bind_group_layouts,device_mismatch')
+  .desc(
+    `
+    Tests createPipelineLayout cannot be called with bind group layouts created from another device
+    - two layouts from same device
+    - two layouts from different device
```

Comment:
any particular reason for two layouts? Could be just one, may make the test shorter. same question for the others tests that use two objects

Reply:
ok, so only one of the two layouts would be from a mismatching device? i.e. cases would be:
- both layouts from correct device
- one from correct device, one from wrong device

?

> The first valid object should skip the validation of the second one.

not sure what you mean by this

Reply:
Yes, your understanding above is correct. 

> The first valid object should skip the validation of the second one.

Missing a **not**,  what I wanted to say was **should not**.  The bindGroupLayouts I wanted in the tests is `bindGroupLayouts: [validLayout, mismatchedLayout]`, not `bindGroupLayouts: [mismatchedLayout, validLayout]`.

Reply:
SGTM. Maybe you want to test both [valid, mismatched] and [mismatched, valid] if we want to try to robustly detect kinds of bugs.